infra/bridge/test/exchange-rate.js

```diff
@@ -27,28 +26,33 @@ describe('exchange rate poller', () => {
   })
 
   it('should fetch exchange rate from API if not cached', async () => {
-    nock('https://api.cryptonator.com')
-      .get('/api/ticker/YYY-USD')
-      .reply(200, { success: true, ticker: { price: '678.9' } })
+    nock('https://api.coingecko.com')
+      .get('/api/v3/exchange_rates')
+      .reply(200, {
+        rates: {
+          btc: { name: 'Bitcoin', unit: 'BTC', value: 1.0, type: 'crypto' },
+          usd: { name: 'US Dollar', unit: '$', value: 11874.946, type: 'fiat' }
+        }
+      })
 
     const response = await request(app)
-      .get('/utils/exchange-rate?market=YYY-USD')
+      .get('/utils/exchange-rate?market=BTC-USD')
```

Comment:
Since the mobile app queries DAI and ETH exchange rate, I'd suggest testing both for 'DAI-USD' and 'ETH-USD' here, in order to catch any regression in the future.

Reply:
I can do that, the reason I chose BTC is because all the values in the api call are returned in BTC value and I have to convert to USD. But I can add DAI and ETH to this test as well.